### ISSUE TITLE
Growth figures seem overestimated

### ISSUE THREAD

**Issue by kennethkichan** [antoinevulcain/Financial-Modeling-Prep-API#32]
I noticed some of the growth rates look exceptionally large, did anything change?
https://financialmodelingprep.com/api/v3/financial-statement-growth/fb
https://financialmodelingprep.com/api/v3/financial-statement-growth/tsla

**Comment by antoinevulcain** [comment]
which field?

**Comment by antoinevulcain** [comment]
@kennethkichan yes we updated all key metrics financial growth ratios they are now up to date

**Comment by kennethkichan** [comment]
Please see below, seem they exceed 100% and in some cases 20000%
Revenue Growth (per Share)
Operating CF Growth (per Share)
Shareholders Equity Growth (per Share)
Net Income Growth (per Share)
Example, https://financialmodelingprep.com/api/v3/financial-statement-growth/fb

**Comment by antoinevulcain** [comment]
for the 10Yr/3Yr for example?

10yr -> (70697000000.0 - 777000000.0) / 70697000000.0 = 0.98900 right?

**Comment by antoinevulcain** [comment]
So yes there is a problem since we are showing 8.76439847243

**Comment by kennethkichan** [comment]
No worries, please let us know when it's updated! Thank you for providing an excellent API, i'm a frequent user!

**Comment by antoinevulcain** [comment]
Sure! thanks for letting me know 👍

**Comment by antoinevulcain** [comment]
Ok so I get it:

Here is the calculation it is a growth per share:

it is (revenue 2019 / number of share 2019) - (revenue 2009 / number of share 2009) / (revenue 2009 / number of share 2009) 

this is why you get this kind of number: example AAPL:

10yr Revenue ->

((260174000000 / 4648913000) - (65225000000 / 924712000)) / (65225000000 /924712000)

-> (55.96 - 70.54) / 70.54

-> -0.2066

This is because of the 10Y Revenue Growth **-> (per Share)** which correspond of the share of the period of the revenue.

Does it makes sense?

**Comment by kennethkichan** [comment]
Right, i get you because it is per share. 

Would you be adding a compound growth figure in the near future?

**Comment by antoinevulcain** [comment]
Yes that’s definitely something doable.
